src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,13 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery lottery = new Lottery();
+        Ball[] ball = new Ball[3];
+        for (int i = 0; i < 3; i++) {
+            ball[i] = lottery.getRandomBall();
+        }
+        for (Ball balls : ball) {
```

Comment:
```suggestion
        for (Ball ball : balls) {
```
логічніше буде трошки підправити тут неймінг